python/lsst/atmospec/__init__.py

```diff
@@ -20,7 +20,7 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 import pkgutil
```

Comment:
@mfisherlevine We don't usually put `pkgutil` imports at this level so I'm wondering whether it should be here given there aren't any external subpackages in the lsst.atmospec namespace.

Reply:
Fair enough, and looks like it's already merged, so 👍

Reply:
I merged the lsstimport change yes. I wasn't sure what your intent was with the pkgutil line. I think it's only needed if you are going to have atmospec_x and atmospec_y packages.

Reply:
None - it was almost certainly cargo-culted that way.